assets/scripts/storage.js

```diff
@@ -0,0 +1,53 @@
+// Saves new goals
+function save() {
+	chrome.storage.sync.get('accomplishGoalsList', function(response){
+		var existingGoals = response.accomplishGoalsList;
+		if(typeof response.accomplishGoalsList === 'undefined') {
+			existingGoals = [];
+		}
+		existingGoals.push({
+		'title': document.getElementsByName('newgoal-title')[0].value,
+		'begin': document.getElementsByName('newgoal-begin')[0].value,
+		'end': document.getElementsByName('newgoal-end')[0].value,
+		'description': document.getElementsByName('newgoal-description')[0].value
+		});
+		chrome.storage.sync.set({'accomplishGoalsList': existingGoals});
+	});
+}
+
+
+// Initialize method that checks if previous goals have been defined and
+// loads them if they have been
+function initialize() {
+	chrome.storage.sync.get('accomplishGoalsList', function(response){
+		if(typeof response.accomplishGoalsList !== 'undefined') {
+			response.accomplishGoalsList.forEach(function(goal) {
+
+				var titleTag = document.createElement('p');
```

Comment:
Could we port all this over to jQuery for code style consistency?

Reply:
Resolved